tutorials/Covid19_MSA_Group_Comparison.md

```diff
@@ -1,3 +1,181 @@
+
+4. Please press ***continue*** to be redirected to workflUX.
+
+## **Step 2: Login with Your ELIXIR Account** 
+
+![welcome screenshot](../screenshots/4.01_tutorial_covid19_welcome.png) 
+
+Now that you have registered, you may log in.
+
+1. Please press the ***login / register*** button again.
+
+You will be forwarded to ELIXIR AAI.
+ 
+![info screenshot](../screenshots/4.04_tutorial_covid19_elixir_provide_info.png) 
+
+2. Please provide workflUX access to the requested information.
+
+## **Step 3: Create a New Job**
+
+After login, additional options will appear in the top bar.
+
+1. Please click on the ***Create Job*** button.
+
+![create job screenshot](../screenshots/4.05_tutorial_covid19_create_job.png) 
+
+2. Select the MSA comparison workflow in the left panel.
+
+![create job screenshot](../screenshots/4.06_tutorial_covid19_create_job.png) 
+
+In the first section, you will be informed with a workflow description.
+
+![title screenshot](../screenshots/4.07_tutorial_covid19_title.png) 
+
+> **Please Note**: This description has been **directly extracted from the CWL workflow** upon import. 
+
+3. Please choose a title for your analysis job.
+
+> **Please Note**: You may submit **batches of runs** at once. This greatly simplifies the specification of large sample sets.
+
+For this demo, we will leave the batch submission disabled.
+
+scroll down
+
+3. To provide parameters, please select ***HTML form***.
+
+![html screenshot](../screenshots/4.08_tutorial_covid19_html.png) 
+
+## **Step 4: Provide Input Parameters**
+
+An HTML form will appear that asks you to provide input parameters for this run. There are only **two required parameters (fasta_1 and fasta_2)**. The **remaining parameters are optional** and pre-set with their defaults, but feel free to play around.
+
+1. Provide the S-protein sequences for the two patient cohorts by copy & pasting following HTTP URLs:
+
+![fasta screenshot](../screenshots/4.09_tutorial_covid19_fasta.PNG) 
+
+![parameter screenshot](../screenshots/4.10_tutorial_covid19_provide_parameter.png) 
+
+> **Please Note**: Your parameters are **automatically validated** to avoid bad surprises when starting the run.
+
+> **Please Note**: The entire parameter form has been created from the information parsed from the CWL workflow itself. No additional configuration was needed upon import.
+If you don‘t know the meaning of a parameter, please cilck on the :information_source: button to get documentation.
+
+scroll down
+
+2. Click on ***validate and create job***.
+
+![validate screenshot](../screenshots/4.11_tutorial_covid19_validata_and_create.png) 
+
+Please wait
+
+3. Once a green success message occurs, you may click on ***Job Execution & Results*** in the top bar. 
+
+![execute screenshot](../screenshots/4.12_tutorial_covid19_job_execution.png) 
+
+## **Step 5: Submit to WES**
+
+Now it is time to submit your job to a WES endpoint for execution.
```

Comment:
This will only work if there is a WES endpoint connected to workflUX, we might need to adapt this section mentioning that you can choose from the execution profiles existing for the workflUX instance.